### ISSUE TITLE
Warning: moveSharedElement is required

### ISSUE THREAD

**Issue by govem** [xotahal/react-native-motion#21]
I'm getting a warning 

`Warning: Failed context type: The context `moveSharedElement` is marked as required in `SharedElement`, but its value is `undefined`.`

when I use the SharedElement.. what value should I give to that property?

**Comment by nandorojo** [comment]
I'm also getting this.

**Comment by rahul13agrawal** [comment]
@xotahal are we going to get response?

**Comment by okechukwu0127** [comment]
Getting same issue.. No solution ?

**Comment by SamiChab** [comment]
Same here... Help please

**Comment by nandorojo** [comment]
I recommend checking out this library: https://github.com/IjzerenHein/react-native-shared-element

I also wrote a guide: https://gist.github.com/nandorojo/e84e938568a018e044aadae9d64456ec

**Comment by SamiChab** [comment]
@nandorojo Thank you so much!! This library is also available for `react-navigation v5`: https://github.com/IjzerenHein/react-navigation-shared-element/blob/navigation-v5/README.md

**Comment by nandorojo** [comment]
Yup! Also, if you are looking to use react native web with react navigation, check out my library [`expo-next-react-navigation`](https://github.com/nandorojo/expo-next-react-navigation)

**Comment by 2xSamurai** [comment]
> I recommend checking out this library: https://github.com/IjzerenHein/react-native-shared-element
> 
> I also wrote a guide: https://gist.github.com/nandorojo/e84e938568a018e044aadae9d64456ec

This one is good. I also managed to get it working. I wasted a lot of time to make 'react-native-motion' work. But I couldn't. Thankfully I came by your comment. Thanks again.

**Comment by nandorojo** [comment]
@2xSamurai Glad to hear it!
